en/core-libraries/email.rst

```diff
@@ -464,17 +464,19 @@ following::
     {
         public function welcome($user)
         {
-            $this->_email->profile('default');
-            $this->_email->to($user->email);
-            $this->_email->subject(sprintf('Welcome %s', $user->name));
+            $this
+                ->to($user->email);
+                ->subject(sprintf('Welcome %s', $user->name))
+                ->template('welcome_mail') // By default template with same name as method name is used.
+                ->layout('custom');
         }
 
         public function resetPassword($user)
         {
-            $this->_email->profile('default');
-            $this->_email->to($user->email);
-            $this->_email->subject('Reset password');
-            $this->set(['token' => $user->token]);
+            $this
+                ->to($user->email);
+                ->subject('Reset password');
+                ->set(['token' => $user->token]);
```

Comment:
Either separate statements because of the trailing semicolons:

```` php
$this->to($user->email);
$this->subject('Reset password');
$this->set(['token' => $user->token]);
````

Or removing the semicolons:

```` php
$this->
    to($user->email)
    subject('Reset password')
    set(['token' => $user->token]);
````
Same goes for the related DocBlock:
https://github.com/cakephp/cakephp/pull/7311/files#diff-0d502f6f5b61a7cd1890732527bd7069R37